src/blue.ts

```diff
@@ -134,7 +132,16 @@ export function blueProxyFactory(env: MembraneBroker) {
             freeze(this);
         }
         get(shadowTarget: BlueShadowTarget, key: PropertyKey, receiver: BlueObject): RedValue {
-            return env.getBlueValue(ReflectGet(this.target, key, env.getRedValue(receiver)));
+            /**
```

Comment:
just a big comment about what's going on here, so in the future we can understand the issue of syncing with the shadowTarget better